.travis.yml

```diff
@@ -36,9 +36,9 @@ install:
 
 script:
   - ./run-qa-checks
-  - pytest --cov=openwisp_controller
+  - pytest --cov=openwisp_controller -c tests/pytest.ini
   - coverage run -a --source=openwisp_controller runtests.py
-  - pytest -c pytest.sampleapp.ini
+  - pytest -c tests/pytest.sampleapp.ini
```

Comment:
Moving these files is a workaround, for some reason, pytest-django is not able to find `manage.py`; [`py` had a new release recently](https://pypi.org/project/py/#history), I suspect that introduced a bug.

Reply:
Looked into recent release changes and possible bugs introduced by `pytest`, `pytest-django` or `py`, none of them seem to be the culprit!

Reply:
these changes are not related to this PR.
I'm getting an error regarding this problem on the current master without this patch.
Please create another patch to solve this problem.

Reply:
Fixed in https://github.com/openwisp/openwisp-controller/pull/224
Please rebase on the current master

Reply:
Thank you, I was working on this last night, I think it's only a workaround so I was still looking for the cause of the error; removed it since it's fixed in master.